backend/protocol_rpc/endpoints.py

```
@@ -796,6 +800,19 @@ def send_raw_transaction(
+        # Send value at transaction submission
+        if genlayer_transaction.type != TransactionType.SEND:
+            try:
+                accounts_manager.update_account_balance(
+                    address=genlayer_transaction.from_address,
+                    value=-value if value else None,
+                )
+            except ValueError as e:
+                raise JSONRPCError(
+                    message=str(e),
+                    data={"sender_address": genlayer_transaction.from_address},
+                )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)